src/System.Text.Json/tests/Serialization/TestClasses.SimpleTestClassWithSimpleObject.cs

```diff
@@ -44,6 +45,7 @@ public class SimpleTestClassWithSimpleObject : ITestClass
                 @"""MyDouble"" : 2.2," +
                 @"""MyDecimal"" : 3.3," +
                 @"""MyDateTime"" : ""2019-01-30T12:01:02.0000000Z""," +
+                @"""MyGuid"" : ""5BB9D872-DA8A-471E-AA70-08E19102683D""," +
```

Comment:
Is it interesting to use GUIDs in some of the other string formats supported by Guid.Parse?

Reply:
The underlying reader/writer only support the default format at the moment ('D' in this case), both for parsing and formatting. The same applies to DateTime(Offset). The serializer/deserializer leverage the reader/writer for converting. Support for custom formats supported by the `Utf8Formatter`/`Utf8Parser` is out of scope for 3.0 and I believe requires the user to create their own converter.

https://github.com/dotnet/corefx/blob/5cc736198cd5d1663a4af6c4e31cccf06ab3735d/src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.TryGet.cs#L627

Reply:
`D` includes lower case but it's not a big deal.

Reply:
> `D` includes lower case

Good point.